doc/man3/SSL_CTX_set_record_padding_callback.pod

```diff
@@ -0,0 +1,49 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_record_padding_callback, SSL_set_record_padding_callback - install callback to specify record padding
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void SSL_CTX_set_record_padding_callback(SSL_CTX *ctx, void (*cb)(SSL *s, int type, size_t len));
+ void SSL_set_record_padding_callback(SSL *ssl, void (*cb)(SSL *s, int type, size_t len));
```

Comment:
The sig for the callbacks here has a void return type - which does match what is actually in the code.